source/patterns/@aws-solutions-constructs/core/lib/vpc-helper.ts

```diff
@@ -44,13 +44,8 @@ export function buildVpc(scope: Construct, props: BuildVpcProps): ec2.IVpc {
 
   let cumulativeProps: ec2.VpcProps = props?.defaultVpcProps;
 
-  if (props?.userVpcProps) {
-    cumulativeProps = overrideProps(cumulativeProps, props?.userVpcProps);
-  }
-
-  if (props?.constructVpcProps) {
-    cumulativeProps = overrideProps(cumulativeProps, props?.constructVpcProps);
-  }
+  cumulativeProps = consolidateProps(cumulativeProps, props?.userVpcProps);
+  cumulativeProps = consolidateProps(cumulativeProps, props?.constructVpcProps);
```

Comment:
consolidateProps takes 3 arguments, the third being construct props, so this can be a single line:

```
  cumulativeProps = consolidateProps(cumulativeProps, props?.userVpcProps, props?.constructVpcProps);
```

Let's look to see if I missed this elsewhere - handling all three props sources (default, client and construct) was the primary driver for creating this routine.